pkg/utils/gardener/secretsrotation/serviceaccounts.go

```diff
@@ -57,7 +57,7 @@ func CreateNewServiceAccountSecrets(ctx context.Context, log logr.Logger, c clie
 	log.Info("ServiceAccounts requiring a new token secret", "number", len(serviceAccountList.Items))
 
 	var (
-		limiter = rate.NewLimiter(rate.Limit(rotationQPS), rotationQPS)
```

Comment:
Oops, I didn't know that `rotationQPS` is also used here for the `ServiceAccount` rotation. Maybe the old name was better after all. Or we could use dedicated constants.
Anyway, it doesn't really matter much 🙈

Reply:
@rfranzke, Will you adapt `reencryptionQPS` to `rotationQPS`?

Reply:
done